src/uct/ib/rdmacm/rdmacm_def.h

```diff
@@ -26,8 +26,8 @@ typedef struct uct_rdmacm_iface   uct_rdmacm_iface_t;
 typedef struct uct_rdmacm_ep      uct_rdmacm_ep_t;
 
 typedef struct uct_rdmacm_priv_data_hdr {
-    uint8_t length;     /* length of the private data */
-    int8_t  status;
+    uint8_t      length;     /* length of the private data */
+    ucs_status_t status;
```

Comment:
can we keep it as uint8_t to make sure it's 1 byte on the wire?
also seems there is missing UCS_S_PACKED here
@alinask fyi

Reply:
reverted